### PR TITLE
newlisp 10.7.0

### PR DESCRIPTION
**Pull request by nijikon**

### CODE REVIEW

**Comment by apjanke** [comment]
Patch is failing to apply to the new version. Going to need manual review.

```
==> Patching
patching file guiserver/newlisp-edit.lsp
Hunk #1 FAILED at 1.
Hunk #2 FAILED at 157.
Hunk #3 FAILED at 225.
3 out of 3 hunks FAILED -- saving rejects to file guiserver/newlisp-edit.lsp.rej
Error: Failure while executing: /usr/bin/patch -g 0 -f -p1
```

**Comment by nijikon** [comment]
I noticed that I pushed a weird version of the change, sorry for that.

File: Library/Formula/newlisp.rb [hunk 1]
```diff
@@ -16,17 +11,16 @@ class Newlisp < Formula
...
-
   depends_on "readline" => :recommended
 
   def install
     # Required to use our configuration
     ENV.append_to_cflags "-DNEWCONFIG -c"
 
+    # fix the prefix in a source file
+    inreplace "guiserver/newlisp-edit.lsp", "#!/usr/local/bin/newlisp", "#!/usr/bin/env newlisp"
+    inreplace "guiserver/newlisp-edit.lsp", "/usr/local/bin/newlisp", "HOMEBREW_PREFIX/bin/newlisp"
```
Comment (bfontaine) [inline Library/Formula/newlisp.rb]: Use `#{opt_bin}` instead of `HOMEBREW_PREFIX/bin` (I assume that’s a typo and you wanted to use `#{HOMEBREW_PREFIX}`).

**Comment by nijikon** [comment]
Can this be merged now?

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/6c3b523212deea7589acb2d7029b86f8bc1ab0f9. Thanks again @nijikon!

Just a nit for future but when editing the same file twice with an `inreplace` you can use a block like this: https://github.com/Homebrew/homebrew/commit/03c925589896962938c2b1f6fec641895b0d737c.

**Pull request closed without merging**